locales/templates/app.pot

```diff
@@ -144,6 +144,22 @@ msgctxt "collection_separator_shots"
 msgid "%1$d shots"
 msgstr ""
 
+#. String displayed when text recognition has an unknown error
+msgctxt "detail_ocr_error_failed"
+msgid "Recognize failed"
```

Comment:
This is still wrong (it should say "Recognition"). Maybe even "Text recognition failed" to avoid confusion.